src/network/networkd-link.c

```diff
@@ -1092,6 +1092,16 @@ int link_address_remove_handler(sd_netlink *rtnl, sd_netlink_message *m, void *u
         return 1;
 }
 
+static int link_set_bridge_vlan(Link *const link) {
```

Comment:
*const? the ptr shall be const? that doesn't really gain us much, and we generally don't place that there since it has no effect on the caller as the ptr itself is copied anyway...